src/app/components/product-owner/menu/menu.component.css

```diff
@@ -1,5 +1,9 @@
-button{
+button, a{
```

Comment:
Those are really generic selectors, are you sure it's safe to style those elements like that, instead of creating reusable classes?